lib/WUI/http/handler_defaults.c

```diff
@@ -0,0 +1,60 @@
+#include "handler.h"
+
+#include "../wui_REST_api.h"
+#include "../wui_api.h"
+#include "../wui.h"
+
+#define GET_WRAPPER(NAME)                                                                       \
+    static void handler_##NAME(struct HttpHandlers *unused, char *buffer, size_t buffer_size) { \
+        (void)unused;                                                                           \
+        NAME(buffer, buffer_size);                                                              \
+    }
+GET_WRAPPER(get_printer);
+GET_WRAPPER(get_version);
+GET_WRAPPER(get_job);
+GET_WRAPPER(get_files);
+#undef GET_WRAPPER
```

Comment:
/question/ As I understand it, the `lib/WUI/http` directory contains implementation of a generic http client/server (which shouldn't have any knowledge of our specific endpoints). In such case, wouldn't it be cleaner to move this file under `lib/WUI` and name it – for example – `wui_handlers.c`?

Reply:
Well, the file structure around WUI is its own chapter (some parts are in `src`, some in `lib`…) and it's not even fully true that the http server would be fully agnostic of the content (the POST part ☹). But I guess this is as good time as any to help it a bit.